packages/react/src/components/error-summary/error-summary.tsx

```diff
@@ -74,7 +78,7 @@ export const ErrorSummary: VoidFunctionComponent<ErrorSummaryProps> = ({
     ));
 
     return (
-        <SectionalBanner
+        <StyleSectionalBanner
```

Comment:
I think we are using "Styled" with a "d" in the other components. so _StyledSectionalBanner_